combat-trainer.lic

```diff
@@ -2259,16 +2261,33 @@ class AttackProcess
       game_state.loaded = false
       waitrt?
     elsif game_state.loaded
-      actions = game_state.next_aim_action.split(';')
-      actions.each do |action|
-        first_word = action.split(' ')[0].downcase
-        matches = [/^You /]
-        matches = ['You get', 'You draw out', 'You pick up', "You're already holding that"] if @get_actions.include?(first_word)
-        matches = ['Roundtime',"You aren't close enough to attack"] if @rt_actions.include?(first_word)
-        matches = ['You put', 'You sheathe'] if @stow_actions.include?(first_word)
-        bput(action.strip, matches)
-        pause 0.25
-        waitrt?
+      if game_state.use_offhand_trainables?
+        skill = game_state.determine_offhand_weapon
+        skill = 'Brawling' if game_state.current_weapon_skill.eql?'Bow'
+        if skill.eql?'Brawling'
+          3.times{bput('gouge left', 'You')}
+          return
+        end
+
+        weap = game_state.wield_offhand_weapon(skill)
```

Comment:
If skill is Light Thrown I think it should lob, otherwise it should jab.

Reply:
I think it should do both? We kind of get double bang for our buck?

Reply:
I could definitely make different action arrays based on the skill containing throw.

Reply:
Not everyone will train LT (even though they should) so they would waste time lobbing at something they couldn't hit.  Another side effect would be if they have super high LT but are doing Xbow in a backhunt.  LT would murder things

Reply:
Fair points! So, how do we want to handle this?

Reply:
Wait, this is why I have a trainables array. They simply don't include Light Thrown.

Reply:
Not me.  I use bola, throwing hammer, and javelin.  There's a penalty for using non-specifically-throwable items.  You can lob the nightstick, but you're getting an additional penalty